### PR TITLE
Fix jdbc tests.

### PR DESCRIPTION
**Pull request by velioglu**
That PR fixes given JDBC tests by replacing the prefix of GUC variables from citusdb to citus.

### CODE REVIEW

File: jdbc/JDBCReleaseTest.java [hunk 1]
```diff
@@ -17,11 +17,10 @@ public static void main(String[] args) throws SQLException {
...
 			test_no_2();
-			//test_no_3();
-			//test_no_4();
-			test_no_6();
-			
-			//test_no_7();
+			test_no_3();
+			test_no_4();
+            test_no_6();
+			test_no_7();
```
Comment (mtuncer) [inline jdbc/JDBCReleaseTest.java]: can you also add ```simplePreparedTestXXX`` to here too. Looks like there are 4 of them.

Comment (velioglu) [inline jdbc/JDBCReleaseTest.java]: Added.

File: jdbc/JDBCReleaseTest.java [hunk 2]
```diff
@@ -17,11 +17,15 @@ public static void main(String[] args) throws SQLException {
...
 			test_no_1();
 			test_no_2();
-			//test_no_3();
-			//test_no_4();
-			test_no_6();
-			
-			//test_no_7();
+			test_no_3();
+			test_no_4();
+            test_no_6();
```
Comment (lithp) [inline jdbc/JDBCReleaseTest.java]: Is the indentation wrong here?

Comment (velioglu) [inline jdbc/JDBCReleaseTest.java]: Silly tab-space difference. I fixed it.

**Comment by lithp** [comment]
I don't know who originally wrote the JDBC tests, maybe @metdos can tell us who's qualified to look at this PR. But assuming nobody steps up in the next few days feel free to merge, this is better than what's currently there :)

**Comment by mtuncer** [comment]
@lithp It was me who originally wrote this file. This was meant to be a trial to see how jdbc queries work with citus and ended up being here few months back.

We definitely need to do a major re-design for this one.

**Comment by lithp** [comment]
@mtuncer Should that re-design block this PR?

**Comment by mtuncer** [comment]
no, this should be good to go

**Comment by velioglu** [comment]
> no, this should be good to go

Does that mean 🚢 ?

**Comment by mtuncer** [comment]
more like 🛳

**Pull request merged**